### PR TITLE
[FEATURE] Portuguese Translation

### PR DESCRIPTION
**Pull request by Alexthegib**

### CODE REVIEW

File: lib/l10n/app_pt.arb [hunk 1]
```diff
@@ -0,0 +1,229 @@
+"about": "Sobre",
```
Comment (BrightDV) [inline lib/l10n/app_pt.arb:1]:
```suggestion
{
    "about": "Sobre",
```

**Review by BrightDV — CHANGES_REQUESTED** [review]
Thanks for PR, just a little mistake.

**Comment by Alexthegib** [comment]
See if its solved

**Comment by BrightDV** [comment]
It doesn't look like... Otherwise, I can do it myself if you can't
Also, if you want, you can translate the [Fastlane folder,](https://github.com/BrightDV/BoxBox/tree/main/fastlane/metadata/android) It is for the F-Droid users.

**Comment by BrightDV** [comment]
It looks the same on my side, I will merge it and then modify it. Thanks a lot!

**Review by BrightDV — APPROVED** [review]

**Comment by Alexthegib** [comment]
> It looks the same on my side, I will merge it and then modify it. Thanks a lot!

On my side it's changed weird

**Pull request merged**